app/src/main/res/navigation/nav_graph.xml

```diff
@@ -298,20 +298,21 @@
         android:name="org.mozilla.fenix.library.bookmarks.selectfolder.SelectBookmarkFolderFragment"
         android:label="@string/bookmark_select_folder_fragment_label"
         tools:layout="@layout/fragment_select_bookmark_folder">
-        <argument
-            android:name="folderGuid"
-            app:argType="string"
-            app:nullable="true" />
         <action
             android:id="@+id/action_bookmarkSelectFolderFragment_self"
             app:destination="@id/bookmarkSelectFolderFragment" />
         <action
             android:id="@+id/action_bookmarkSelectFolderFragment_to_bookmarkAddFolderFragment"
             app:destination="@id/bookmarkAddFolderFragment" />
         <argument
-            android:name="visitedAddBookmark"
+            android:name="allowCreatingNewFolder"
             android:defaultValue="false"
             app:argType="boolean" />
+        <argument
+            android:name="hideFolderGuid"
+            android:defaultValue="null"
```

Comment:
I don't think this is actually `null` I think it's setting the default value to "null". Try `@null`.

Reply:
Ah, thanks! It's right in the docs, too 👍 

So I've tried it both ways, and in both cases (`null` and `@null`) I get a real `null` value, not a "null" string. But let's go with what the docs say.